action.yml

```diff
@@ -0,0 +1,90 @@
+name: 'Setup IOG librararies'
+description: 'Installs libsodum, libsecp256k1 and libblst'
+inputs:
```

Comment:
dead

Reply:
Right. We probably won't need this anymore. I wasn't sure in the beginning.